src/jit/jitconfigvalues.h

```diff
@@ -218,6 +218,10 @@ CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 0)
 CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 1)
 #endif // !defined(DEBUG) && !defined(_DEBUG)
 
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+CONFIG_INTEGER(JitMinOptsTrackGCrefs, W("JitMinOptsTrackGCrefs"), 0) // Allow GC roots to be tracked w/MinOpts
```

Comment:
clrconfigvalues.h is for things read by the VM; jitconfigvalues.h is for things read by the VM. Note that clrconfigvalues.h does duplicate many things, *if* those things are used by JIT32/JIT64, which does not have jitconfigvalues.h and that mechanism. That is usually what causes confusion.

Yes, there will be code dynamically executed because of this, but presumably it will be very small/cheap.